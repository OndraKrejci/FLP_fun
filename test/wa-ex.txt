T,A,S,B
a,b
T
T->ASB
T->SB
T->AS
S->ASB
S->SB
S->AS
A->aAS
A->a
A->aS
B->SbS
B->bb
B->aAS
B->a
B->aS