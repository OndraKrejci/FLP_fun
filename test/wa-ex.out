T,A,S,B,<SB>,<AS>,<bS>,a',b'
a,b
T
T->A<SB>
T->SB
T->AS
S->A<SB>
S->SB
S->AS
A->a'<AS>
A->a
A->a'S
B->S<bS>
B->b'b'
B->a'<AS>
B->a
B->a'S
<SB>->SB
<AS>->AS
<bS>->b'S
a'->a
b'->b